src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -949,7 +949,7 @@ void Transformations::MainSnippets(void) {
 #if defined(OPENVINO_ARCH_X86_64)
     auto is_supported_matmul = [this](const std::shared_ptr<const ov::Node>& n) {
         const auto matmul = ov::as_type_ptr<const ov::op::v0::MatMul>(n);
-        if (!matmul || matmul->is_dynamic())
+        if (!matmul)
```

Comment:
Only fp32 dynamic brgemm is supported - could you please add the following code in L960:
```
        // Only FP32 Brgemm is supported
        if (matmul->is_dynamic())
            return false;
```

Reply:
Good catch, thanks! Corrected